hack/single-node/bootkube-up

```diff
@@ -39,3 +39,6 @@ echo
 echo "Bootstrap complete. Access your kubernetes cluster using:"
 echo "kubectl --kubeconfig=cluster/auth/kubeconfig get nodes"
 echo
+echo "To copy logs to the host run:"
+echo "../scripts/vagrant-copylogs"
```

Comment:
I don't think we need to add these to the vagrant examples

Reply:
removed